js/binance.js

```diff
@@ -533,7 +533,7 @@ module.exports = class binance extends Exchange {
 
     async withdraw (currency, amount, address, params = {}) {
         let response = await this.wapiPostWithdraw (this.extend ({
-            'asset': currency,
+            'asset': this.commonCurrencyCode (currency),
```

Comment:
This one probably needs a reverse-mapping.

Reply:
Actually I've made a custom `commonCurrencyCode` too for binance but for some reason I missed it from the commit;

```js
```